addons/stock/static/src/js/inventory_validate_button_controller.js

```diff
@@ -65,12 +59,31 @@ var InventoryValidationController = ListController.extend({
         }
 
         prom.then(function () {
-            self._rpc({
-                model: 'stock.inventory',
-                method: self.validateMethod,
-                args: [self.inventory_id],
-                context: self.context
-            }).then(function (res) {
+            var rpc = Promise.resolve();
+            if (self.context && self.context.button_validate_picking_ids) {
+                // Clean context to remove default inventory values during move and move lines
+                // creation.
+                var context = Object.keys(self.context || {})
+                    .filter(key => ! (key.startsWith('default_')))
+                    .reduce((obj, key) => {
+                      obj[key] = self.context[key];
+                      return obj;
+                    }, {});
```

Comment:
It isn't more simple to do only one loop ?
```suggestion
                var context = {};
                for (const key in self.context) {
                    if (!key.startWith('default_')) {
                        context['key'] = self.context['key'];
                    }
                }
```